Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/security.yml

```diff
@@ -77,3 +77,25 @@
       amount: 2
     - id: MagazineRifle
       amount: 4
+
+- type: entity
+  parent: GunSafe
+  id: GunSafeRocketLauncher
+  name: RPG safe
+  components:
+  - type: StorageFill
+    contents:
+    - id: WeaponLauncherRocket
```

Comment:
for all of these amount: 1 is the default and can be removed

Reply:
Use a table and give it 20% probability for extra rocket.